CAVEATS.md

```diff
@@ -0,0 +1,52 @@
+# Platform specific notes
+
+## Windows
+
+### Spectest requirements
+
+You need recent version of either Strawberry Perl or ActiveState Perl.
+
+If you are working with ActiveState Perl you need the Mingw gcc compiler.
+
+You need msys git installed and you need "\Program Files\Git\cmd" on your
+execution path and NOT "\Program Files\Git\bin".
+
+You need a win32 curl program.
+
+
+## macOS
+
+### Dynamic libraries
+
+If Rakudo is installed into $HOME directory (this is also the case when
+rakubrew <https://rakubrew.org/> is used) then some modules may fail to
+install due to missing dynamic libraries. This could be caused by a
+situation when the library is not provided by macOS system itself but
+installed using Homebrew or MacPorts package managers. In this case the
+library cannot be loaded by rakudo binary. This is a security precaution
+taken by modern versions of macOS. The precaution works by restricting the
+directories available for loading dynamic libraries from only to the
+system-predefined set and those where the application binary is actually
+located. For example, if "rakudo" binary is installed under "$HOME/raku/bin"
+then aside of the system-wide locations it can access libraries in
+"$HOME/raku/lib" only.
+
+For the above stated reason, if a problem with availability of a dynamic
+library occurs then creating a symlink to the library in one of the
+following locations could help:
+
+- <rakudo binary location path>/../lib
+- $HOME/lib (should be created if doesn't exists yet)
+
+The latter is one of the allowed system locations.
+
+Also, sometimes the library version provided by macOS itself is older than a
+module expects. This could result in erratic behaviors of the module. In
+such a case a solution would be to install a package with newer version of
+the library and also create symlinks into one of the two locations.
```

Comment:
Maybe add an example?

Reply:
The example would depend on macOS version. Besides, I can only think of libssl issues @coke had. But there is no full certainty that it was the case.